src/main/kotlin/no/nav/familie/oppdrag/avstemming/AvstemmingSenderMQ.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.familie.oppdrag.avstemming
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.jms.JmsException
+import org.springframework.jms.core.JmsTemplate
+import org.springframework.stereotype.Service
+import java.lang.UnsupportedOperationException
+
+@Service
+class AvstemmingSenderMQ(val jmsTemplateAvstemming: JmsTemplate,
+                         @Value("\${oppdrag.mq.enabled}") val erEnabled: String) {
+
+    fun sendGrensesnittAvstemming() {
+        if (!erEnabled.toBoolean()) {
```

Comment:
Kunne dette vært en feature toggle?

Reply:
Med Unleash, tenker du? Ja, men da må vi sette opp den integrasjonen. Kan ta det i en separat PR hvis det er stemning for det :)

Reply:
Jepp! Egen PR er nice 👍